pallets/liquid-staking-v2/src/lib.rs

```diff
@@ -149,32 +182,18 @@ mod pallet {
     #[pallet::getter(fn current_era)]
     pub type CurrentEra<T: Config> = StorageValue<_, EraIndex, ValueQuery>;
 
-    /// Records relay operations during each era.
+    /// In-flight/Succeeded/Failed staking operation history.
     #[pallet::storage]
-    pub type UnbondingOperationHistory<T: Config> =
-        StorageMap<_, Twox64Concat, EraIndex, Operation<BlockNumberFor<T>, BalanceOf<T>>>;
-
-    /// Record all the pending unstaking requests.
-    /// Key is the owner of assets.
-    #[pallet::storage]
-    #[pallet::getter(fn account_unstake)]
-    pub type AccountUnstake<T: Config> = StorageDoubleMap<
+    #[pallet::getter(fn staking_operation_history)]
+    pub type StakingOperationHistory<T: Config> = StorageDoubleMap<
```

Comment:
This means we should record bond/unbond response either, but we are not going to add `record_bond` and `record_unbond` calls. So what do you think about that?

Reply:
this can be useful because sometimes we may need to check if `bond/unbond` is already done on relaychain, this status can be useful I think, so I think maybe we should have `record_status` call etc to record them